init/systemd/00-internal0.link

```diff
@@ -0,0 +1,6 @@
+[Match]
+OriginalName=*
+
+[Link]
+Name=internal0
```

Comment:
What is this `internal0`, and why a name like this was selected? Is there some standard naming for systemd?